versioned_docs/version-5.x/use-linking.md

```diff
@@ -4,7 +4,7 @@ title: useLinking
 sidebar_label: useLinking
 ---
 
-The `useLinking` hook lets us handle deep links in our apps.
+The `useLinking` hook lets us handle deep links in our apps. You probably want to use the [`linking` prop on `NavigationContainer`](navigation-container.md#linking) instead of using this hook directly.
```

Comment:
Why does this hook even exist? Does it handle some specific scenario that `linking` prop can't handle?

Reply:
It was how deep linking worked previously. The `linking` prop is a new API and will be the recommended way, but we can't remove `useLinking` since it'll be a breaking change.

Reply:
maybe we should say that it's deprecated